apps/costcontrol/test/unit/startup_test.js

```diff
@@ -280,12 +279,34 @@ suite('Application Startup Modes Test Suite >', function() {
         assertFTEStarted(applicationMode, done);
```

Comment:
Explain why the assert is before the dispatch. Say something like "this install a listener in charge of assert the correct blah blah blah" or something like that.